workflow/design.md

```diff
@@ -0,0 +1,28 @@
+I am taking an example of PyPi(Python package installer) and Pycharm(IDE designed specifically for python project development).
```

Comment:
Can you further refine the design document and move it to that location `docs/design/ide_kpm_workflow.md` when you have free time? We are about to close the Pretest issue and welcome the official LFX task.

Reply:
Yeah, Okay I'll do it today.